activiti-cloud-services-query/activiti-cloud-services-query-rest/src/main/java/org/activiti/cloud/services/query/events/handlers/ProcessVariableUpdateEventHandler.java

```diff
@@ -34,6 +34,9 @@ public void handle(ProcessVariableEntity updatedVariableEntity) {
         BooleanExpression predicate = QProcessVariableEntity.processVariableEntity.name.eq(variableName)
                 .and(
                         QProcessVariableEntity.processVariableEntity.processInstanceId.eq(String.valueOf(processInstanceId))
+                )
+                .and(
+                        QProcessVariableEntity.processVariableEntity.markedAsDeleted.eq(Boolean.FALSE)
```

Comment:
As discussed previously, let's stop using `markedAsDeleted` flag and use real deletes.